### DISCUSSION TITLE
home assistant core integration

### DISCUSSION THREAD

**Discussion by enry86cami** [blakeblackshear/frigate#9983] [Config Support]
### Describe the problem you are having

I installed Frigate on a LXC container.
I have a home assistant core installation, in HA I installed frigate integration and configured it with the frigate ip : http://192.168.0.27:5000/.
If I try to type the following link on the browser:
http://ccab4aaf-frigate-fa:5000/api/stats
http://ccab4aaf-frigate:5000/api/stats
192.168.0.27-fa:5000

The browser says: Is not possible to reach the website…

### Version

0.13.2

### Frigate config file

```yaml
mqtt:
  host: 192.168.0.9
  user: xxx
  password: xxxx

cameras:
  Garage: # <--- this will be changed to your actual camera later
    enabled: True
    ffmpeg:
      inputs:
        - path: rtsp://admin:xxx@192.168.0.22:554/cam/realmonitor?channel=1&subtype=0
      hwaccel_args: preset-vaapi
          #roles:
          #  - detect

go2rtc:
  streams:
    Garage:
      - rtsp://admin:xxxx@192.168.0.22:554/cam/realmonitor?channel=1&subtype=0
```

### Relevant log output

```shell
home assistant log:
Error fetching information from http://ccab4aaf-frigate-fa:5000/api/stats: Cannot connect to host ccab4aaf-frigate-fa:5000 ssl:default [Name or service not known]
Error fetching information from http://ccab4aaf-frigate:5000/api/stats: Cannot connect to host ccab4aaf-frigate:5000 ssl:default [Name or service not known]
Error fetching information from http://192.168.0.27-fa:5000/api/stats: Cannot connect to host 192.168.0.27-fa:5000 ssl:default [Name or service not known]
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

CPU (no coral)

**Comment by danuals** [comment]
you can access it from a browser on http://192.168.0.27:5000/.?

**Reply by enry86cami** [reply]
Yes

**Comment by danuals** [comment]
I believe these addresses are for the ha installed frigate .
http://ccab4aaf-frigate-fa:5000/api/stats)

my install is on a different server to HA and I use the usual ip for setting up. Same one used via browser. 

what’s it you want to do. 
it’s the frigate proxy add on your using?
